Lombiq.HelpfulLibraries/Libraries/Utilities/StringExtensions.cs

```diff
@@ -22,5 +22,23 @@ public static class StringExtensions
         /// </summary>
         public static bool ContainsLoose(this string? text, string? toFind) =>
             text != null && toFind != null && text.Contains(toFind, StringComparison.InvariantCultureIgnoreCase);
+
+        /// <summary>
+        /// A shortcut for <c>string.Contains(string, StringComparison.OrdinalIgnoreCase)</c>.
+        /// </summary>
+        public static bool ContainsOrdinalIgnoreCase(this string? text, string? value) =>
+            text != null && !string.IsNullOrEmpty(value) && text.Contains(value, StringComparison.OrdinalIgnoreCase);
+
+        /// <summary>
+        /// A shortcut for <c>string.StartsWith(string, StringComparison.OrdinalIgnoreCase)</c>.
+        /// </summary>
+        public static bool StartsWithOrdinalIgnoreCase(this string? text, string? value) =>
+            text != null && !string.IsNullOrEmpty(value) && text.StartsWith(value, StringComparison.OrdinalIgnoreCase);
+
+        /// <summary>
+        /// A shortcut for <c>string.Equals(string, StringComparison.OrdinalIgnoreCase)</c>.
+        /// </summary>
+        public static bool EqualsOrdinalIgnoreCase(this string? text, string? value) =>
+            text != null && !string.IsNullOrEmpty(value) && text.Equals(value, StringComparison.OrdinalIgnoreCase);

```

Comment:
It looks to me that `"".EqualsOrdinalIgnoreCase("")` will return `false` which seems counter-intuitive.

Reply:
Good point. Dávid, please fix this too.

Reply:
Then I think others like `"".StartsWithOrdinalIgnoreCase("")` should be true as well?

Reply:
Yep.

Reply:
![image](https://user-images.githubusercontent.com/164360/128510555-6c250f1c-7af7-4382-be3b-477db5214f0a.png)

Reply:
done